tests/calc/utilities/poscar/test_modify_vacuum.py

```diff
@@ -54,12 +58,56 @@ def test_modify_vacuum_gap_absolute(self, test_slab, vacuum_gap_size):
         assert modified_slab.vacuum_gap == pytest.approx(vacuum_gap_size)
         assert modified_slab.thickness == pytest.approx(slab.thickness)
 
-    @pytest.mark.parametrize('vacuum_gap_size', [1.0])
+    @parametrize('vacuum_gap_size', [1.0])
     @infoless
-    def test_modify_vacuum_gap_too_small(self, test_slab, vacuum_gap_size):
+    def test_gap_too_small(self, test_slab, vacuum_gap_size):
         """Test the ModifyVacuumCLI class with absolute vacuum gap size."""
         parser = ModifyVacuumCLI().parser
         slab, *_ = test_slab
         args = parser.parse_args([str(vacuum_gap_size), '-a'])
         with pytest.raises(SystemExit):
             modified_slab = ModifyVacuumCLI().process_slab(slab, args)
+
+    def test_parse_cli_args_absolute_negative_gap(self):
+        """Test that parse_cli_args raises for a negative absolute gap."""
+        parser = ModifyVacuumCLI()
+        with pytest.raises(SystemExit):
+            parser.parse_cli_args(['-1.0', '--absolute'])
+
+    @parametrize('vacuum_gap_size', [-1.0, -5.0])
+    @infoless
+    def test_negative_gap_raises(self, test_slab, vacuum_gap_size):
+        """Check complaints when the vacuum gap size is negative."""
+        slab, *_ = test_slab
+        vacuum_gap_info = VacuumGapInfo(size=vacuum_gap_size, absolute=True)
+        with pytest.raises(VacuumError):
+            modified_slab = modify_vacuum.modify_vacuum(slab, vacuum_gap_info)
+
+    @infoless
+    def test_not_enough_vacuum_error(self, test_slab):
+        """Test that NotEnoughVacuumError is raised correctly."""
+        slab, *_ = test_slab
+        gap = VacuumGapInfo(size=0.5, absolute=True, accept_small_gap=False)
+        with pytest.raises(VacuumError):
+            modified_slab = modify_vacuum.modify_vacuum(slab, gap)
+
+    @infoless
+    def test_accept_small_gap(self, test_slab):
+        """Test that NotEnoughVacuumError is raised correctly."""
+        slab, *_ = test_slab
+        if slab.thickness <= 1e-8:  # monolayer; would lead to zero volume cell
+            return
```

Comment:
```suggestion
        if slab.thickness <= 1e-8:
            pytest.skip('Single layer; would lead to zero volume cell')
```

Nicer so that we don't get a "passing" test but a "skipped" one. Then one can more easily find out why it was skipped, without going into the code (with `pytest -rS tests/...`)

Reply:
Already done in b96e0fdab7ccf51d298a6d16f99edb2aa87b04a8